Sources/Plasma/Apps/plClient/plClient.cpp

```diff
@@ -140,8 +139,15 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "pfConsole/pfConsole.h"
 #include "pfConsole/pfConsoleDirSrc.h"
 #include "pfConsoleCore/pfConsoleEngine.h"
+#if defined(PLASMA_PIPELINE_DX)
+    #include <d3d9.h>
```

Comment:
I'd prefer to see this being included by `plDXPipeline.h` if forward declarations aren't enough :)